src/api/ledger/parse/suspended-payment-cancel.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+'use strict';
+const assert = require('assert');
+const utils = require('./utils');
+
+function parsePaymentMemos(tx) {
+  if (!Array.isArray(tx.Memos) || tx.Memos.length === 0) {
+    return undefined;
+  }
+  return tx.Memos.map((m) => {
+    return utils.removeUndefined({
+      type: m.Memo.parsed_memo_type,
+      format: m.Memo.parsed_memo_format,
+      data: m.Memo.parsed_memo_data
+    });
+  });
+}
+
+function parsePayment(tx: Object): Object {
+  assert(tx.TransactionType === 'SuspendedPaymentCancel');
```

Comment:
```
SuspendedPaymentCancel 
SusPayCancel
```

I recall during discussions of this that @justmoon requested the non abbreviated form was present in the txn json (as seen here with the TransactionType). I think in the general sense, he wanted any public apis to use SuspendedPayment rather than SusPay.  

It probably doesn't really matter, but I went out of my way to [patch](https://github.com/ripple/rippled/pull/1224) rippled in accordance with Stefan's wishes, so I figured I'd bring it up again.